executor/executor.go

```diff
@@ -138,9 +138,11 @@ func doRunSetupCommand(emitter screwdriver.Emitter, f *os.File, r io.Reader, set
 	return nil
 }
 
-func doRunCommand(guid, path string, emitter screwdriver.Emitter, f *os.File, fReader io.Reader) (int, error) {
+func doRunCommand(guid, path string, emitter screwdriver.Emitter, f *os.File, fReader io.Reader, stepName string) (int, error) {
 	executionCommand := []string{
 		"export SD_STEP_ID=" + guid,
+		// escape not necessary because step name is limited to [A-Za-z0-9_-]
```

Comment:
This is described here: https://github.com/screwdriver-cd/data-schema/blob/b7cf7ab0bc5039570aeb03daa790f0d67e183544/config/job.js#L92-L103